src/main/java/org/linksharing/server/user/UserProfileDetails.java

```diff
@@ -1,11 +1,30 @@
 package org.linksharing.server.user;
 
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
 import lombok.Data;
 
 @Data
+@Entity(name = "user_profiles")
 public class UserProfileDetails {
```

Comment:
Needs to be one-to-one relationship with `users`

Reply:
I'll implement it using tips from this article.
https://www.baeldung.com/jpa-one-to-one